azure-core/azure/core/paging.py

```diff
@@ -33,14 +33,13 @@
 from typing import Dict, Any, List, Callable, Optional, TYPE_CHECKING  # pylint: disable=unused-import
 
 if TYPE_CHECKING:
-    from .pipeline import HttpResponse  # pylint: disable=unused-import
```

Comment:
We are using HttpResponse in our type annotations.

Reply:
we don't have type annotations yet for this file. Will add it back in #5022